x-pack/plugin/logstash/src/test/java/org/elasticsearch/xpack/logstash/action/DeletePipelineRequestTests.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.logstash.action;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+public class DeletePipelineRequestTests extends AbstractWireSerializingTestCase<DeletePipelineRequest> {
```

Comment:
Can you make these request/response tests implement `mutateInstance` as well? That's required to automatically check `.equals()` for the case of an object of the same type that's not equal.

Reply:
Done!